### PR TITLE
add deployment from resources file

### PR DESCRIPTION
**Pull request by iakov-aws**
*Description of changes:*
add deployment from resources file (only Datasets and Dashboards are supported)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by darken99 — CHANGES_REQUESTED** [review]
Why not self.loadf_resources() in run() ?

File: cid/common.py [hunk 1]
```diff
@@ -225,8 +226,21 @@
             logger.debug(f"Issue logging action {action}  for dashboard {dashboard_id} , due to a urllib3 exception {str(e)} . This issue will be ignored")
 
 
+    def load_resources(self):
+        ''' load additional resources from command line parameters
+        '''
+        if get_parameters().get('resources'):
+            fileneme = get_parameters().get('resources')
+            resources = yaml.safe_load(open(fileneme))
```
[the remark this answers is not in the document]
Comment (darken99) [reply cid/common.py:234]: with open(get_parameters().get('resources'), 'r', encoding='utf-8') as filename:

**Comment by iakov-aws** [comment]
> Why not self.load_resources() in run() ?
We do not have these parameters in run as they go AFTER command. if we will use it in run the syntax will be something like

cid-cmd --resources aaa.yaml deploy --dashboard-id my-cudos

this will be super confusing for cusomer

File: cid/common.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (iakov-aws) [reply cid/common.py:234]: agree

[the remark this answers is not in the document]
Comment (iakov-aws) [reply cid/common.py]: done

**Review by darken99 — APPROVED** [review]

**Pull request merged**